src/command/http-command.ts

```diff
@@ -256,25 +280,40 @@ export class HttpCommand implements CommandInterface<HttpOptions> {
 		await pStream;
 	}
 
+	private toJson(input: Output): OutputJson {
```

Comment:
Let's call this `outputToJson` otherwise it's dangerously close to the standard `toJSON` method and seems like it serializes the command object itself.

Reply:
Alternatively, if we make the returned object a class, it can implement `toJSON()`, which gets called automatically as needed.

Reply:
The measurement by itself is not an instance, we just execute a method within it - might be worth changing that behaviour at some point.

For now, i just named the method `toJsonOutput`.